docker/user.sh

```diff
@@ -52,6 +52,28 @@ then
     mkdir --parents "$HOME"/output
 fi
 
+# Prepare the python virtual environment. We copy the mounted directory if it exists to `.python_venv` to fix the symbolic
```

Comment:
nit: I think it's not clear to me what virtual environment we are preparing here, is that the virtual environment where Macaron is installed and run, or is it the virtual environment provided from the user.